xrdsst/controllers/base.py

```diff
@@ -19,6 +20,38 @@ class Meta:
             (['-v', '--version'], {'action': 'version', 'version': BANNER})
         ]
 
+    config_file = "config/base.yaml"
+    config = None
+    api_key_default = "X-Road-apikey token=<API_KEY>"
+    api_key_id = {}
+
+    def create_api_key(self, roles_list, config, security_server):
+        self.log_info('Creating API key for security server: ' + security_server['name'])
+        roles = '[\\"'
```

Comment:
This ``json`` acrobatics can probably made much simpler by creating Python list of strings and then converting that to json with ``json.dumps``

```
Python 3.8.5 (default, Jul 28 2020, 12:59:40) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import json
>>> roles = [ "role1", "role2" ]
>>> roles.append("roleN")
>>> json.dumps(roles)
'["role1", "role2", "roleN"]'
```

```